fdbserver/VersionedBTree.actor.cpp

```diff
@@ -2772,7 +2772,8 @@ struct RedwoodRecordRef {
 			case 1:
 				return *(int32_t*)r;
 			case 2:
-				return (((int64_t)((int48_t*)r)->high) << 16) | (((int48_t*)r)->low & 0xFFFF);
+				return ((int64_t) static_cast<uint32_t>(reinterpret_cast<const int48_t*>(r)->high) << 16) |
+				       (((int48_t*)r)->low & 0xFFFF);
```

Comment:
@sfc-gh-satherton Is this going away on release-7.0 as well?

Reply:
Yes.